azure-functions-maven-plugin/pom.xml

```diff
@@ -195,11 +194,6 @@
             <artifactId>reflections</artifactId>
             <version>${reflections.version}</version>
         </dependency>
-        <dependency>

```

Comment:
Why are we deleting this reference and how is it related to the problem we want to solve in this change list?

Reply:
We upgraded the azure sdk version to 1.14.0 in library module in the previous PR(#302) which import a  reference to com.fasterxml.jackson.core version 2.9.4 and it conflicts with the version we indirectly referenced through azure-storage.

It causes the functions module build failure and directs us to the problem that we have duplicate dependencies in library module and in functions module. So we remove the dependency in functions module to solve the build failure.

Reply:
This answers why. But it does not relate to the topic of THIS specific change list.
The problem should be addressed in #302 or a follow-up PR dedicated to it.

Reply:
Yes, this is not related to this change list. I separate it to PR #304